### PR TITLE
Disable -Wno-conversion on MSVC compiler

### PR DESCRIPTION
**Pull request by jonahryandavis**

### CODE REVIEW

**Comment by jonahryandavis** [comment]
@johnkslang Can you take a look at this PR? Sorry, I'm not familiar with the process for getting eyes on pull requests in this repository.

**Comment by jonahryandavis** [comment]
For context, this is breaking glslang's roll into ANGLE: https://chromium-review.googlesource.com/c/angle/angle/+/2435250

File: BUILD.gn [hunk 1]
```diff
@@ -76,7 +76,9 @@ if (!defined(glslang_angle)) {
 
 config("glslang_public") {
   include_dirs = [ "." ]
-  cflags = [ "-Wno-conversion" ]
+  if (!is_win || is_clang) {
```
Comment (ShabbyX) [inline BUILD.gn:79]: Does it really need !is_win? Clang should accept this flag regardless of platform.

Comment (jonahryandavis) [reply BUILD.gn:79]: gcc also can accept this flag, this is the way it's done in other parts of the gn files

**Pull request merged**